### PR TITLE
Lpjg2cmor

### PR DESCRIPTION
**Pull request by janitm**
The functionality for CMORizing LPJ-Guess datafiles is ready to be incorporated into the master branch. The majority of the new feature takes place in the new file lpjg2cmor.py

### CODE REVIEW

File: examples/ifsmon.py [hunk 1]
```diff
@@ -17,12 +17,13 @@
...
 variables = {"Amon" : ["tas","uas","vas"]}
-startdate = datetime.date(1990,1,1)
+startdate = datetime.date(1870,1,1)
 interval = relativedelta(months=1)
 #srcdir = os.path.dirname(os.path.abspath(ece2cmorlib.__file__))
 #datadir = os.path.join(srcdir,"test","test_data","ifsdata","3hr")
 mydir = os.path.dirname(os.path.abspath(__file__))
-datadir = os.path.join(mydir,"..","test","test_data","ifsdata","3hr")
+#datadir = os.path.join(mydir,"..","test","test_data","ifsdata","3hr")
+datadir = os.path.join("/Users/anthoni-p/EC-Earth/supermuc/runs/r1902-merge-new-components_impi/LNTS/run_i108_n0_micro_phase2_10years/output/ifs/001")
```
Comment (goord) [inline examples/ifsmon.py]: We would rather have an example that works on the included test data.

File: examples/ifsmon.py [hunk 2]
```diff
@@ -17,12 +17,13 @@
...
 
 variables = {"Amon" : ["tas","uas","vas"]}
-startdate = datetime.date(1990,1,1)
+startdate = datetime.date(1870,1,1)
 interval = relativedelta(months=1)
 #srcdir = os.path.dirname(os.path.abspath(ece2cmorlib.__file__))
 #datadir = os.path.join(srcdir,"test","test_data","ifsdata","3hr")
 mydir = os.path.dirname(os.path.abspath(__file__))
-datadir = os.path.join(mydir,"..","test","test_data","ifsdata","3hr")
+#datadir = os.path.join(mydir,"..","test","test_data","ifsdata","3hr")
```
Comment (goord) [inline examples/ifsmon.py]: See comment below

File: examples/ifsmon.py [hunk 3]
```diff
@@ -17,12 +17,13 @@
 logging.basicConfig(level=logging.DEBUG)
 
 variables = {"Amon" : ["tas","uas","vas"]}
-startdate = datetime.date(1990,1,1)
+startdate = datetime.date(1870,1,1)
```
Comment (goord) [inline examples/ifsmon.py]: See below

File: examples/ifsmon.py (hunk 1 above)
Comment (janitm) [reply examples/ifsmon.py]: Apologies for these, I should have checked and undone any changes not related to the LPJ-Guess CMORization before making the pull request

Comment (goord) [reply examples/ifsmon.py]: No problem, we fix the examples when we resolve issue #223

**Pull request merged**